packages/core/src/configuration/Configuration.spec.ts

```diff
@@ -0,0 +1,104 @@
+import 'reflect-metadata';
+
+import { Configuration } from './Configuration';
+import { EnvCredentialProvider } from '../credentials-provider';
+import { Credentials } from '../credentials-provider/CredentialsProvider';
```

Comment:
Import can be shortened
```suggestion
import { Credentials } from '../credentials-provider';
```